web/ui/react-app/src/pages/Labels.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge } from 'reactstrap';
+
+interface LabelProps {
+  value: any;
+}
+
+const Labels: FC<RouteComponentProps & LabelProps> = ({ value }) => {
+  const [showMore, doToggle] = useState(false);
+
+  const toggleMore = () => {
+    doToggle(!showMore);
+  };
+
+  return (
+    <>
+      <Button size="sm" color="primary" onClick={toggleMore}>
+        {showMore ? 'More' : 'Less'}
+      </Button>
+      {showMore ? (
+        <>
+          <div>
+            {Object.keys(value).map((_, i) => {
+              return (
+                <div key={i} className="row inner-layer">
+                  <div className="col-md-6">
+                    {i === 0 ? <div className="head">Discovered Labels</div> : null}
+                    <div>
+                      {Object.keys(value[i].discoveredLabels).map((labelName, j) => (
```

Comment:
Object entries will give you the key and value as a pair as first arg to the map callback ```map([key, val].....)```. Then this ```{`${labelName}="${value[i].discoveredLabels[labelName]}"`}```
can be shortenet to ```{`${key}="${val}"`}```